lib/routes/routes_generator.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/routes/routes.dart';
+import 'package:workoutpersonalizer_frontend/ui/creators_corner_page.dart';
+import 'package:workoutpersonalizer_frontend/ui/dashboard_page.dart';
+import 'package:workoutpersonalizer_frontend/ui/my_workouts_page.dart';
+
+class RouteGenerator {
+  static Route<dynamic> generateRoute(RouteSettings settings) {
+    switch (settings.name) {
+      case routeDashboard:
+        return MaterialPageRoute(builder: (_) => DashboardPage());
+      case routeCreatorsCorner:
+        return MaterialPageRoute(builder: (_) => CreatorsCornerPage());
```

Comment:
Also not sure if this is just me but I'm seeing duplicates on DevTools!

[Screen Shot 2021-11-11 at 5 44 28 PM](https://user-images.githubusercontent.com/54044854/141394455-f76a93c7-d41e-4f51-93ea-820707d110a8.png)

Reply:
Yeah everytime I click on a nav bar item, it seems to be creating a new page and adding it to expanded. Good catch Lily